tests/model_fields/test_jsonfield.py

```diff
@@ -596,12 +596,24 @@ def test_usage_in_subquery(self):
             self.objs[3:5],
         )
 
+    def test_array_key_contains(self):
+        tests = [
+            ([], [self.objs[7]]),
+            ("bar", [self.objs[7]]),
+            # Oracle falls back to pattern-based containment checking for non-dict rhs.
+            ("ar", [self.objs[7]] if connection.vendor == 'oracle' else []),
```

Comment:
This workaround is not necessary with my proposition
```suggestion
            ('ar', []),
```

Reply:
Yep. Much better :+1: